atomicfu-maven-plugin/build.gradle.kts

```
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2017-2018 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+plugins {
+    `maven-publish`
+    id("kotlin-jvm-conventions")
+}
+
+dependencies {
+    compileOnly(libs.kotlin.stdlib)
+    api(project(":atomicfu-transformer"))
+    api(libs.bundles.maven)
+}
+
+val outputDir = tasks.compileKotlin.get().destinationDirectory
+
+publishing.publications {
+    named<MavenPublication>("maven").configure {
+        pom.configureMavenPluginPomAttributes(project, outputDir.get().asFile.path)
+        pom.packaging = "maven-plugin"
+    }
+}
+
+val mavenUserHome = System.getProperty("maven.user.home")
+val mavenRepoLocal = System.getProperty("maven.repo.local")
+
+val generatePomFileForMavenPublication by tasks.getting(GenerateMavenPom::class)
+
+// runs the plugin description generator
+val generatePluginDescriptor by tasks.registering(Exec::class) {
+    dependsOn(generatePomFileForMavenPublication, project(":atomicfu-transformer").tasks.named("publishToMavenLocal"))
```

```suggestion
    dependsOn(generatePomFileForMavenPublication)
    dependsOn(project(":atomicfu-transformer").tasks.named("publishToMavenLocal"))
```

formatting nitpick: put the `dependsOn()` on separate lines, so it's easier to read.